sdk/go/things_test.go

```diff
@@ -441,7 +442,7 @@ func TestUpdateThing(t *testing.T) {
 				ID:       id,
 				Type:     "app",
 				Name:     "test_app",
-				Metadata: "test_metadata2",
+				Metadata: map[string]interface{}{"meta": "data2"},
```

Comment:
why not `metadata2 = map[string]interface{}{"meta": "data2"}` here?

Reply:
I don't understand what you want me to do here, can you explain?

Reply:
Done.